crypto/rand/rand_vxworks.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+ 
+#ifndef _GNU_SOURCE
+# define _GNU_SOURCE
+#endif
+
+#if defined(OPENSSL_SYS_VXWORKS)
+
+#include <openssl/rand.h>
+#include "rand_lcl.h"
+#include "internal/rand_int.h"
+#include "internal/cryptlib.h"
+
+# if defined(OPENSSL_RAND_SEED_OS)    
+#  include <randomNumGen.h>

```

Comment:
Why do you need this file?

Reply:
this is the include for the randomNumGenLib in vxWorks, it contains functions as randBytes()